sql/core/v1.2/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFilters.scala

```diff
@@ -231,37 +229,37 @@ private[sql] object OrcFilters extends OrcFiltersBase {
     // Since ORC 1.5.0 (ORC-323), we need to quote for column names with `.` characters
     // in order to distinguish predicate pushdown for nested columns.
```

Comment:
Since we removed `quoteIfNeeded` in this file completely, I believe we can remove this old comment (231~232) together in both files v1.2(here) and v2.3.